state/state.go

```diff
@@ -34,19 +42,128 @@ type DownloadedObject struct {
 	Object, Filename string
 }
 
+type DynamoDBStater struct {
+	Session          *session.Session
+	Service          string
+	BackfillInterval time.Duration
+}
+
+func NewDynamoDBStater(session *session.Session, service string, backfill int) *DynamoDBStater {
+
+	return &DynamoDBStater{
+		Session:          session,
+		Service:          service,
+		BackfillInterval: time.Hour * time.Duration(backfill),
+	}
+}
+
+// Used for unmarshaling and adding objects to DynamoDB
+type Record struct {
+	S3Object string
+	Time     time.Time
+	TTL      int64
```

Comment:
Could you put a comment here indicating what the TTL data looks like? I'm guessing it's a future date formatted as unix seconds-since-epoch?